File: multiStructure/frameworks/js-playwright/pages/yourCart/yourCartPage.js
class YourCartPage {
    constructor(page) {
      this.page = page;
  
      this.itemPrice = page.locator('.inventory_item_price');
      this.checkoutButton = page.locator('#checkout');
      this.title = page.locator('.title');
    }
  
    async getPrice() {
        const priceText = await this.itemPrice.textContent();
        return parseFloat(priceText.replace('$', ''));
    }
  
    async clickOnCheckoutButton() {
      await this.checkoutButton.click();
    }

    async getTitleText() {
      return await this.title.first().textContent();
    }
  }
  
  module.exports = {YourCartPage };

File: multiStructure/frameworks/csharp-playwright/Pages/YourCartPage.cs
using Microsoft.Playwright;

namespace PlaywrightFramework.Pages;

public class YourCartPage
{
    private readonly IPage _page;

    public YourCartPage(IPage page)
    {
        _page = page;
    }

    private ILocator ItemPrice => _page.Locator(".inventory_item_price");
    private ILocator CheckoutButton => _page.Locator("#checkout");
    private ILocator Title => _page.Locator(".title").First;

    public async Task<double> GetPriceAsync()
    {
        var priceText = await ItemPrice.TextContentAsync();
        return double.Parse(priceText!.Replace("$", ""));
    }

    public async Task ClickOnCheckoutButtonAsync()
    {
        await CheckoutButton.ClickAsync();
    }

    public async Task<string> GetTitleTextAsync()
    {
        return await Title.TextContentAsync() ?? string.Empty;
    }
}